nx-X11/programs/Xserver/GL/mesa/GLcore/Imakefile

```diff
@@ -1,12 +1,7 @@
 XCOMM $XFree86: xc/programs/Xserver/GL/mesa/GLcore/Imakefile,v 1.3 2002/05/31 18:12:36 dawes Exp $
 
 #if DoLoadableServer
-#if !BuildModuleInSubdir
```

Comment:
I remove `BuildModuleInSubdir` because:

* it is not needed for building the GL extension
* it makes the imake structure more complex than necessary
* I plan to remove the  BuildModuleInSubdir stuff completely later on.